day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,38 @@
+class Frog
+  attr_accessor :frogs
+
+  def initialize(frogs)
+    @frogs = frogs
+  end
+
+  def print
+    until @frogs == 0
+      if @frogs == 1
+        @plural = 'frog'
+        @grammar = 'was'
+      else
+        @plural = 'frogs'
+        @grammar = 'were'
+      end
+      puts "#{@frogs} speckled #{@plural} sat on a log\neating some most delicious bugs.\nOne jumped in the pool where its nice and cool,"
+
+      @frogs -= 1
+
+      if @frogs == 1
```

Comment:
You have the same if blocks here as above. How could you refactor to make it so you only have those if blocks once?